src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
+      <div class="stars stars--0 stars__wrapper">
+        <div class="stars stars__star stars__unchecked"></div>
+
+        <div class="stars stars__star stars__unchecked"></div>
+
+        <div class="stars stars__star stars__unchecked"></div>
+
+        <div class="stars stars__star stars__unchecked"></div>
+
+        <div class="stars stars__star stars__unchecked"></div>
```

Comment:
Each star should have only stars__star class and no additional classes. Figure out how to use :nth-child rule in css